### ISSUE TITLE
Using the Install Script has been failing for quite some time now.  

### ISSUE THREAD

**Issue by webmaxdb** [Arachni/arachni#789]
[ncurses.txt](https://github.com/Arachni/arachni/files/550926/ncurses.txt)

This is the output of cli:

## (4/12) ncurses
  * Downloading http://ftp.gnu.org/pub/gnu/ncurses/ncurses-6.0.tar.gz -  0% ETA:                           
  * Extracting
  * Cleaning
  * Configuring (CFLAGS=-fPIC ./configure --prefix=/root/Desktop/arachni-build-dir/arachni/system/usr)
  * Compiling
Build failed, check /root/Desktop/arachni-build-dir/arachni/build/logs/ncurses for details.
When you resolve the issue you can run the script again to continue where the process left off.

**Comment by Zapotek** [comment]
It's not an install script, it's a build script and it's not really aimed at users, you should be using the stable packages or the nightlies if you need to use the cutting edge.

Also, it's been working on my build boxes, but to be honest I've been having trouble on my dev machines, I think a GCC upgrade broke something at some point.
The build boxes use older distros in order to build against an older version of `libc` (and GCC and accompanying build tools) in order to support running on old distros.

So for their purpose (building release and nightly packages) they're working fine, for anything else use at your own risk.

I am periodically checking for a fix though because I'd like for them to work everywhere, but haven't found one yet.

I guess I'm partly at fault for the confusion regarding their purpose, I should stop mentioning them at the `experimental` branch [README](https://github.com/Arachni/arachni/tree/experimental#source) file.

Btw, the build scripts have their own issue tracker: https://github.com/Arachni/build-scripts/issues

**Comment by Zapotek** [comment]
Excellent timing to ping me about this, I just made another attempt at searching for a solution and this time it worked.

Another issue came up though, I'll let you know of the final result.

**Comment by Zapotek** [comment]
The fix is in the `updates` branch: https://github.com/Arachni/build-scripts/tree/updates

I'm close to releasing and I don't want to mess the package up at the last moment.

**Comment by webmaxdb** [comment]
cant wait to hear from you.



Robert Gouin CCENT, MSDN Dev, CEH, DBA, OSCP, OSWP Wifu, CISSP ... xyz
P. 860-837-3747 - F. 855-593-9333
http://www.webmaxdb.com
rgouin@webmaxdb.com

**Comment by Zapotek** [comment]
Ha, beat you to the punch. :P

**Comment by webmaxdb** [comment]
OK that all makes sense. If you need any help pls let me know

**Comment by Zapotek** [comment]
I wouldn't mind some feedback, you can [download](https://github.com/Arachni/build-scripts/archive/updates.zip) it and check it out.

**Comment by webmaxdb** [comment]
ok i can try it out now.  what flavor did you want me to use?  nix windows
mac?


Robert Gouin CCENT, MSDN Dev, CEH, DBA, OSCP, OSWP Wifu, CISSP ... xyz
P. 860-837-3747 - F. 855-593-9333
http://www.webmaxdb.com
rgouin@webmaxdb.com

**Comment by Zapotek** [comment]
That script works only on Linux and OSX so try whatever you've got handy.

**Comment by webmaxdb** [comment]
I am going to try it right now, running debian which script should I run?
 just build.sh?


Robert Gouin CCENT, MSDN Dev, CEH, DBA, OSCP, OSWP Wifu, CISSP ... xyz
P. 860-837-3747 - F. 855-593-9333
http://www.webmaxdb.com
rgouin@webmaxdb.com

**Comment by Zapotek** [comment]
Yep.

**Comment by webmaxdb** [comment]
Already running I will let you know how it fairs.

**Comment by webmaxdb** [comment]
Before it was always erroring out on ncurses and this time it blew right
past it.  Currently on 8 of 12 postgresql.  You want me to send the logs or
anything once it is done?


Robert Gouin CCENT, MSDN Dev, CEH, DBA, OSCP, OSWP Wifu, CISSP ... xyz
P. 860-837-3747 - F. 855-593-9333
http://www.webmaxdb.com
rgouin@webmaxdb.com

**Comment by Zapotek** [comment]
Nah, if arachni runs we're OK.

**Comment by webmaxdb** [comment]
Build successful.

**Comment by Zapotek** [comment]
Excellent, thanks.

**Comment by webmaxdb** [comment]
your build script online is broken again?


Robert Gouin CCENT, MSDN Dev, CEH, DBA, OSCP, OSWP Wifu, CISSP ... xyz
P. 860-837-3747 - F. 855-593-9333
http://www.webmaxdb.com
rgouin@webmaxdb.com

**Comment by Zapotek** [comment]
Are you having issues? Can you provide some more information please?

**Comment by webmaxdb** [comment]
It's the second exact issue as last time before you had me cut.  How about
instead of emailing you I will try it again and write a ticket for when you
have time..

**Comment by Zapotek** [comment]
Yes, please open a ticket at the tracker of the build-scripts with associated error logs and the like.

**Comment by dogasantos** [comment]
wget https://ftp.gnu.org/pub/gnu/ncurses/ncurses-6.0.tar.gz
--2019-08-02 18:28:17--  https://ftp.gnu.org/pub/gnu/ncurses/ncurses-6.0.tar.gz
Resolving ftp.gnu.org (ftp.gnu.org)... 209.51.188.20, 2001:470:142:3::b
Connecting to ftp.gnu.org (ftp.gnu.org)|209.51.188.20|:443... connected.
ERROR: The certificate of ‘ftp.gnu.org’ is not trusted.
ERROR: The certificate of ‘ftp.gnu.org’ is not yet activated.
The certificate has not yet been activated
Converted links in 0 files in 0 seconds.

wget --no-check-certificate https://ftp.gnu.org/pub/gnu/ncurses/ncurses-6.0.tar.gz
--2019-08-02 18:28:39--  https://ftp.gnu.org/pub/gnu/ncurses/ncurses-6.0.tar.gz
Resolving ftp.gnu.org (ftp.gnu.org)... 209.51.188.20, 2001:470:142:3::b
Connecting to ftp.gnu.org (ftp.gnu.org)|209.51.188.20|:443... connected.
WARNING: The certificate of ‘ftp.gnu.org’ is not trusted.
WARNING: The certificate of ‘ftp.gnu.org’ is not yet activated.
The certificate has not yet been activated
HTTP request sent, awaiting response... 200 OK
Length: 3131891 (3.0M) [application/x-gzip]
Saving to: ‘ncurses-6.0.tar.gz’

ncurses-6.0.tar.gz                         100%[=====================================================================================>]   2.99M  2.13MB/s    in 1.4s

2019-08-02 18:28:41 (2.13 MB/s) - ‘ncurses-6.0.tar.gz’ saved [3131891/3131891]
